test/helper-config.ts

```diff
@@ -0,0 +1,11 @@
+export const base_url: string = process.env.BASE_URL ?? " ";
```

Comment:
What if the `BASE_URL` doesn't exists in env?
```suggestion
export const base_url: string = process.env.BASE_URL ?? "<your_base_url>";
```

Same goes for other env variables defined in this file.